Docs: Add more information about Connectors (#95)
aragon/connect: docs/connectors/tokens-app.md

# Tokens App Connector

> This is an Aragon app specific connector for the Tokens app build using The Graph.

Reviewer: ```suggestion
This is an Aragon app specific connector for the Tokens app built using The Graph.
```

## Connector API

> To create a new instance of the connector you need the specific Tokens app address and subgraph URL:

Reviewer 2: ```suggestion
To create a new instance of the connector you need the specific Tokens app address and a subgraph URL:
```

```javascript
import { TokenManager } from '@aragon/connect-thegraph-tokens'

const tokenManager = new TokenManager(
  TOKENS_APP_ADDRESS,
  TOKENS_APP_SUBGRAPH_URL
)
```

> Once you have a an instance of the `TokenManager` object you can use the followig API to fetch data.

Reviewer: ```suggestion
Once you have a an instance of the `TokenManager` object you can use the following API to fetch data.
```

> Once you have a an instance of the `TokenManager` object you can use the following API to fetch data.

Reviewer 2: ```suggestion
Once you have a an instance of the `TokenManager` object, you can use the following API to fetch data.
```

### tokenManager\#token\(tokenManagerAddress\)

> Get the data of a Tokens app.

Reviewer: ```suggestion
Get the data of the token used in the provided Tokens app. 
```

| Name                  | Type             | Description                             |
| :-------------------- | :--------------- | :-------------------------------------- |
| `tokenManagerAddress` | `String`         | Address of the Tokens app.              |
| returns               | `Promise<Token>` | Result data parsed as a `Token` entity. |

### tokenManager\#tokenHolders\(tokenAddress, first, skip\)

> Get the data of the token holders of a Minime token.

Reviewer: ```suggestion
Get the data of the token holders of the token used in the provided app.
```
